### ISSUE TITLE
Cannot run AddRoutes, error about RTMidi

### ISSUE THREAD

**Issue by enzyme69** [JPfeP/AddRoutes#1]
<img width="700" alt="Screen Shot 2020-04-16 at 11 04 40 am" src="https://user-images.githubusercontent.com/3952687/79403413-3a86e480-7fd2-11ea-983c-4caece527bff.png">

**Comment by enzyme69** [comment]
I copy paste the module into Blender app but still this happens

```
Traceback (most recent call last):
  File "/Applications/Blender.app/Contents/Resources/2.82/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/Users/jimmygunawan/Library/Application Support/Blender/2.82/scripts/addons/AddRoutes-master/__init__.py", line 52, in <module>
    from . import midi
  File "/Users/jimmygunawan/Library/Application Support/Blender/2.82/scripts/addons/AddRoutes-master/midi.py", line 38, in <module>
    from rtmidi.midiutil import open_midiinput
ModuleNotFoundError: No module named 'rtmidi.midiutil'

```

**Comment by enzyme69** [comment]
Apparently:
pip3 install python-rtmidi

instead of 
pip3 install rtmidi

then I need to also manually pip3 install
mido
oscpy

**Comment by JPfeP** [comment]
Indeed, you figured out by yourself that some additional libraries are need. 
I made some clarification on the front page.

**Comment by df-0** [comment]
I'm also having trouble with this. I get the following when I try to activate:
```Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 2.93\2.93\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "C:\Users\davef\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\AddRoutes\__init__.py", line 56, in <module>
    from . import data
  File "C:\Users\davef\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\AddRoutes\data.py", line 32, in <module>
    from . import g_vars
  File "C:\Users\davef\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\AddRoutes\g_vars.py", line 22, in <module>
    import rtmidi
  File "C:\Users\davef\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\AddRoutes\rtmidi\__init__.py", line 4, in <module>
    from ._rtmidi import *  # noqa
ModuleNotFoundError: No module named 'rtmidi._rtmidi'

```

I've tried running `pip3 install python-rtmidi` then `pip3 install mido` , `pip3 install oscpy`, it tells me:
```Requirement already satisfied: python-rtmidi in c:\python39\lib\site-packages (1.4.9)```
```Requirement already satisfied: mido in c:\python39\lib\site-packages (1.2.10)```
```Requirement already satisfied: oscpy in c:\python39\lib\site-packages (0.5.0)```

Any ideas how to fix it? I'm on blender 2.93, addroutes 0.32, windows 10

**Comment by rekliner** [comment]
Had same issue as @df-0 .  It's because of python-rtmidi, not addroutes but I was able to resolve:   blender 2.93.4 and 2.93.5 AddRoutes .32 on Win 10.  It seems due to python being updated from 3.6 to 3.9.  I found the same error in the blender console by running 'import rtmidi'.  Had to install Visual Studio C++ 14.  I vaguely remember needing to copy python36.lib to python39.lib ... and was able to pip install rtmidi outside of blender which made the file "_rtmidi.cp39-win_amd64.pyd".

copied "_rtmidi.cp39-win_amd64.pyd" into C:\Program Files\Blender Foundation\Blender 2.93\2.93\python\lib\site-packages\rtmidi

 In the blender console you should be able to install the rtmidi extension now by running these commands:
```
import pip
pip.main(['install', 'rtmidi', '--user'])
pip.main(['install', 'python-rtmidi', '--user'])

```
once the rtmidi dependency is present the addon can be activated.

**Comment by Norman-w** [comment]
I try to remove
 /Applications/Blender.app/Contents/Resources/2.93/python/lib/python3.9/site-packages/rtmidi 
folder
then it works!

**Comment by LeoFdAguiar** [comment]
Hi,

I'm trying to install in Blender 3. I keep getting an error. 

I tried the pip3 installs that enzyme69 did. 

I have a rtmidi folder in my OSX Library. I tried copying and pasting that folder into Blender's Content/Resources.../site-packages folder and still get the error.

I also tried deleting the folder like Norman-w.

When the folder ISN'T present inside Blender's Content/Resources.../site-packages the error I get is ModuleNotFoundError: No module named 'rtmidi'.

When the folder IS present inside Blender's Content/Resources.../site-packages the error I get is
ModuleNotFoundError: No module named 'rtmidi._rtmidi'.
